here-naksha-lib-geo/src/commonMain/kotlin/naksha/geo/BoundingBoxProxy.kt

```diff
@@ -20,8 +20,79 @@ class BoundingBoxProxy() : ListProxy<Double>(Double::class) {
         addAll(arrayOf(west, south, southWestAlt, east, north, northEastAlt))
     }
 
+    @JsName("fromCoord")
+    constructor(coord: ICoordinates) : this() {
+        when (coord) {
+            is PointCoord -> setToPoint(coord.getLongitude(),coord.getLatitude())
+            is MultiPointCoord -> addMultiPoint(coord)
+            is LineStringCoord -> addLineString(coord)
+            is MultiLineStringCoord -> addMultiLineString(coord)
+            is MultiPolygonCoord -> addMultiPolygon(coord)
+            is PolygonCoord -> addPolygon(coord)
+        }
+    }
+
     private fun has(value: Double?): Boolean = value != null && !value.isNaN()
 
+    fun setToPoint(longitude: Double, latitude: Double): BoundingBoxProxy {
```

Comment:
for some reason (not known to me), sometimes it's expected to add buffer around point when creating bbox

Reply:
I would propose then to add a third optional parameter `margin: Double = 0.0` (require it to be greater/equal zero and not being infinite or NaN)